hummingbot/connector/derivative/orderly_perpetual/orderly_perpetual_derivative.py

```
@@ -0,0 +1,1864 @@
+        if not use_exchange_ids or not exchange_order_ids:
+            use_exchange_ids = False
+            # Collect all client_order_ids
+            client_order_ids = [order.client_order_id for order in orders_to_cancel]
+
+        # Get exchange symbols for all trading pairs
+        exchange_symbols = []
+        for trading_pair in symbols:
+            try:
+                symbol = await self.exchange_symbol_associated_to_pair(trading_pair)
+                exchange_symbols.append(symbol)
+            except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)